.travis.yml

```diff
@@ -16,11 +16,11 @@ matrix:
       sudo: required
       dist: trusty
     # As above, Python 3.6, setup.py install, xspec 12.10
-    - env: XSPECVER="12.10.0e" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=2 TRAVIS_PYTHON_VERSION="3.6"
+    - env: XSPECVER="12.10.1b" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=2 TRAVIS_PYTHON_VERSION="3.6"
       sudo: required
       dist: trusty
     # As above, python 3.7, numpy 1.14, matplotlib 2.0
```

Comment:
Can we have 'matplotlib 2.0' changed to 'matplotlib 3' here ;-) Assuming that it works with mpl v3.

Reply:
just did it in #590

Reply:
but I'm looking at #590 and I don't see it in the repository

Reply:
e.g. https://github.com/sherpa/sherpa/pull/590/commits/478f0f49f7b49f6995e120737ddb05fe31ced857

note I'm talking about the comment not the actual version used in the run..

Reply:
sorry, I meant #591

Reply:
Aha - I think you meant #591